main/vcl/inc/win/salids.hrc

```diff
@@ -23,92 +23,89 @@
 #define _SV_SALIDS_HRC
 
 // Cursor
-#define SAL_RESID_POINTER_NULL						10000
-#ifndef W40ONLY
-#define SAL_RESID_POINTER_HELP						10001
```

Comment:
I think a comment would be useful here, telling that ID 10001 was once used for `SAL_RESID_POINTER_HELP`, that was deprecated in December 2025.